uaclient/jobs/update_messaging.py

```diff
@@ -75,6 +83,40 @@ def get_contract_expiry_status(
     return ContractExpiryStatus.ACTIVE, remaining_days
 
 
+@lru_cache(maxsize=None)
+def get_contextual_esm_info_url() -> str:
```

Comment:
Should we add a specifc MOTD test for PRO instances ?

Reply:
yep good idea!

Reply:
done!